docker_fargate/docker_fargate_stack.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, scope: Construct, context: str, env: dict, vpc: ec2.Vpc, **kw
             self,
             f'{stack_prefix}-Service',
             cluster=cluster,            # Required
-            cpu=128,                    # Default is 256
+            cpu=256,                    # Default is 256
```

Comment:
would be good to add a comment referencing [doc to pick matching cpu and memory](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html)